src/cmd/create.js

```diff
@@ -0,0 +1,36 @@
+/* @flow */
+import path from 'path';
+import childProcess from 'child_process';
+
+import promisify from 'es6-promisify';
+import libnpx from 'libnpx';
+
+import {createLogger} from '../util/logger';
+
+const log = createLogger(__filename);
+
+type CreateParams = {
+  projectPath: string,
```

Comment:
Shouldn't this be a required option? If so, it would be easier to pass in as an argument, like:

```
web-ext create my-project-path
```

Reply:
yeah, right now I defined it as an additional yargs option to keep the initial version smaller and easier to evaluate, nevertheless I totally agree that make it just an additional required parameter would be way nicer for the user (and it is basically also as create-webextension works when used on its own).